app/Http/Controllers/AdministrationController.php

```diff
@@ -204,6 +217,41 @@ private static function getNominationsData()
         return $data;
     }
 
+    private static function getResultsData()
+    {
+        $nominees = Nominee::all();
+        $faculties = Faculty::all();
+        $votes = Vote::all();
+        $data = [];
+
+        foreach ($nominees as $nominee) {
+            $nominee['faculty'] = $faculties->find($nominee['faculty_id']);
+            $nominee['votes'] = $votes->where('nominee_id', $nominee['id'])->where('isFake', 0)->count();
+            array_push($data, $nominee);
+        }
+
+        /*usort(
+        $data,
+        function ($a, $b) {
+        return $a['votes'] - $b['votes'];
+        }
+        );*/
+
+        arsort($data);
```

Comment:
Since you are pushing whole objects into data array, resulted array for sorting will look something like this
`
[
{id:..., 
faculty :..., 
votes:... } 
,... 
] `

How can this sort function know, by which parametr it should sort these nested objects? How can you guarantee that it is sorted by votes when there are other properties?